.github/workflows/check-code.yml

```diff
@@ -33,6 +33,13 @@ jobs:
   cargo-clippy:
     name: Run Clippy checks
     runs-on: ubuntu-latest
+    container:
+      image: paritytech/ci-unified:bullseye-1.74.0
```

Comment:
This would need to be a repo-wide variable, which should be updated once we bump our dependencies to a new version, if we want to share it across workflows. I agree it's out of scope, but I cc @ggera to see if there's anything low effort we can do to minimize duplication.